app/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,192 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="appName">RedAlert</string>
+    <string name="signing_up">Signing up for push notifications...</string>
+    <string name="okay">OK</string>
+    <string name="error">Error</string>
```

Comment:
No
_________________________
error = error  👍 
or
error = Fehler   👎 not so great , Even if the correct translation is